monai/deploy/cli/exec_command.py

```diff
@@ -27,7 +27,7 @@ def create_exec_parser(subparser: _SubParsersAction, command: str, parents: List
     parser = subparser.add_parser(command, formatter_class=argparse.HelpFormatter, parents=parents, add_help=False)
 
     parser.add_argument("--input", "-i", help="Path to input folder/file (default: input)")
```

Comment:
Would this change `folder` only, too?

Reply:
This one is for the internal app's input path so we would keep this as it is (it can accept either folder path or file path).